ddl/failtest/fail_db_test.go

```diff
@@ -534,3 +535,20 @@ func (s *testFailDBSuite) TestModifyColumn(c *C) {
 
 	tk.MustExec("drop table t, t1, t2, t3, t4, t5")
 }
+
+func (s *testFailDBSuite) TestPartitionAddPanic(c *C) {
+	defer func() {
+		c.Assert(failpoint.Disable("github.com/pingcap/tidb/ddl/CheckPartitionByRangeErr"), IsNil)
```

Comment:
I think we'd better put it after line547.

Reply:
updated